superset-frontend/spec/helpers/Worker.ts

```diff
@@ -21,9 +21,12 @@ class Worker {
 
   onMessage: Function;
 
+  terminate: Function;
+
   constructor(stringUrl: string) {
     this.url = stringUrl;
     this.onMessage = () => {};
+    this.terminate = () => {};
```

Comment:
Could you explain why is this needed? Not sure how it this related to AsyncAceEditor.

Reply:
During `ReactAce` `willUnmount` cycle the `brace` package invokes `Worker` `terminate`. 

<img width="623" alt="Screen Shot 2021-02-22 at 7 33 04 AM" src="https://user-images.githubusercontent.com/70410625/108696807-cade6180-74e0-11eb-824e-18bf54476d01.png">

Reply:
Thanks for the info! Makes sense.